src/main/res/xml/preferences.xml

```diff
@@ -105,7 +105,7 @@
             android:summary="@string/pref_reader_config_summary"
             android:title="@string/pref_reader_config_title" />
         <EditTextPreference
-            android:defaultValue=""
+            android:defaultValue="pjpires@gmail.com"
```

Comment:
Email has been removed. The application prompts the user to submit logs when deactivated, but by default provides no means to discover an email address to submit them to. I emailed you about this today and assumed you were comfortable with giving users a method to contact you. I have updated the commit and removed the value.